packages/web/docs/src/pages/product-updates/2025-01-08-static-breaking-change-threshold.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Added a Conditional Breaking Change Static Threshold
+description:
+  A minor improvement to our conditional breaking changes behavior that allows changes to be
+  considered breaking based on a minimum number of requests.
+date: 2025-01-08
+authors: [jdolle]
+---
+
+Expect a lot of improvements to the Hive Console (Schema Registry) in 2025.
```

Comment:
I realized we don't use the term "Hive Console" anywhere in our marketing material. So I'll adjust that to only be `Hive Schema Registry`.